tests/cover/test_direct_strategies.py

```diff
@@ -251,6 +252,14 @@ def test_decimals():
     assert find(ds.decimals(), lambda f: f.is_finite() and f >= 1) == 1
 
 
+@pytest.mark.parametrize('b', [10 ** 20, -(10 ** 20)])
+def test_too_large_decimal_bounds(b):
+    with pytest.raises(InvalidArgument):
+        ds.decimals(min_value=b, places=10).example()
+    with pytest.raises(InvalidArgument):
+        ds.decimals(max_value=b, places=10).example()
```

Comment:
A different phrasing: "too large to represent with current precision and requested number of places".

Interpreting this as None would (with the other changes in this pull) mean that we only generate things in part of the bounds, and that's typically a 'we'd test less than you think' error.